### PR TITLE
Sort the files in dir to ensure paired readings

### PR DESCRIPTION
**Pull request by ntadimeti**

### CODE REVIEW

**Comment by ntadimeti** [comment]
Addresses #216

File: atacworks/dl4atac/utils.py [hunk 1]
```diff
@@ -236,7 +236,7 @@ def gather_files_from_cmdline(input, extension):
             res = [path]
         elif os.path.isdir(path):
             paths = [os.path.join(path, f)
-                     for f in os.listdir(path)]
+                     for f in sorted(os.listdir(path))]
```
Comment (avantikalal) [inline atacworks/dl4atac/utils.py:239]: Can we add to the help messages in `scripts/cmd_args.py` to say that the files will be read in sorted order?

**Review by avantikalal — CHANGES_REQUESTED** [review]

File: atacworks/dl4atac/utils.py (hunk 1 above)
Comment (ntadimeti) [reply atacworks/dl4atac/utils.py:239]: done

File: scripts/cmd_args.py [hunk 2]
```diff
@@ -265,7 +269,8 @@ def add_inference_options(parser):
                      model only outputs denoised tracks and \
                      this option becomes irrelevant.')
     parser.add('--tracks', action='store_true',
-               help='Output denosied tracks from atacworks. If --task is classification, \
+               help='Output denosied tracks from atacworks. If --task \
```
Comment (avantikalal) [inline scripts/cmd_args.py:272]: denosied -> denoised

**Review by avantikalal — APPROVED** [review]

**Pull request merged**